src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const revers = {};
```

Comment:
```suggestion
  const reversedRobot = {};
```
Would be great to give more meaningful names to variables